Ical.Net/Utility/DateUtil.cs

```diff
@@ -24,6 +24,9 @@ public static IDateTime EndOfDay(IDateTime dt)
     public static DateTime GetSimpleDateTimeData(IDateTime dt)
         => dt.Value;
 
+    public static CalDateTime AsCalDateTime(this DateTime t)
```

Comment:
I remember, that's what you mentioned lately :)

Reply:
Exactly. In this PR its particularly helpful, because it works well with the null-propagating operator.